index.js

```diff
@@ -1,4 +1,16 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import EresourceSearch from './src/EresourceSearch';
+import EresourceSearchModal from './src/Modal';
 
-export default props => <EresourceSearch {...props} />;
+function PluginFindEresource(props) {
+  return (props.justDisplayModal ?
+    <EresourceSearchModal {...props} /> :
+    <EresourceSearch {...props} />);
+}
+
+PluginFindEresource.propTypes = {
+  justDisplayModal: PropTypes.bool,
```

Comment:
Also can we change the naming to a defaultOpen prop instead of justDisplayModal? Or is this a naming convention we are following elsewhere?

Reply:
In an earlier version I did call this `openByDefault` (following the naming of the `<Accordion>` component's `closedByDefault` prop). But I consciously chose a different name because parallel is inexact and misleading. What's happening here is not just starting the modal in the open state — so that when it's closed, the trigger reappears — it's doing a fundamentally different thing, allowing the client code to control the open/close cycle. So I think the current name is better: more explicit, precise and accurate.